### PR TITLE
KOGITO-4922 - Update monitoring metrics

### PR DESCRIPTION
**Pull request by r00ta**
Kogito-runtime pr: https://github.com/kiegroup/kogito-runtimes/pull/1237

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: kogito-travel-agency/extended/travels/src/test/java/org/acme/travel/TravelIT.java [hunk 1]
```diff
@@ -98,7 +98,8 @@ public void testProcessMetrics() {
                 .get("/metrics")
                 .then()
                 .statusCode(200)
-                .body(containsString("kie_process_instance_running_total{app_id=\"default-process-monitoring-listener\",process_id=\"travels\",} 1.0"));
+                .body(containsString(
+                        "kie_process_instance_running_total{app_id=\"default-process-monitoring-listener\",artifactId=\"travels\",process_id=\"travels\",version=\"2.0.0-SNAPSHOT\",} 1.0"));
```
Comment (danielezonca) [inline kogito-travel-agency/extended/travels/src/test/java/org/acme/travel/TravelIT.java:102]: Please avoid explicit 2.0.0-SNAPSHOT

File: kogito-travel-agency/extended/visas/src/main/java/org/kie/kogito/app/ProcessEventListenerConfig.java [hunk 2]
```diff
@@ -18,27 +18,33 @@
...
+import org.kie.kogito.monitoring.core.common.MonitoringRegistry;
 import org.kie.kogito.process.impl.DefaultProcessEventListenerConfig;
 
 @ApplicationScoped
 public class ProcessEventListenerConfig extends DefaultProcessEventListenerConfig {
 
     private VisaApplicationPrometheusProcessEventListener listener;
 
+    @Inject
+    ConfigBean configBean;
```
Comment (danielezonca) [inline kogito-travel-agency/extended/visas/src/main/java/org/kie/kogito/app/ProcessEventListenerConfig.java:34]: This is not ideal, it is probably better if possible to hide a similar detail from user but I don't have a good idea for now 🤔

**Comment by r00ta** [comment]
Jenkins retest this please

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/ProjectMetadataProvider.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    private static final String projectVersion;
+    private static final String projectArtifactId;
+
+    static {
+        String propertyFileName = "project.properties";
+        try {
+            props.load(GrafanaDockerComposeIT.class.getClassLoader().getResourceAsStream(propertyFileName));
+            projectVersion = props.getProperty("project.version");
+            projectArtifactId = props.getProperty("project.artifactId");
```
Comment (danielezonca) [inline dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/ProjectMetadataProvider.java:33]: Can you also please verify that value returned is not `null` in both cases and that it doesn't start with `${`? This last check is to make sure maven filtering is properly configured

File: dmn-drools-springboot-metrics/src/test/java/org/kie/kogito/examples/ProjectMetadataProvider.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    private static final String projectVersion;
+    private static final String projectArtifactId;
+
+    static {
+        String propertyFileName = "project.properties";
+        try {
+            props.load(GrafanaDockerComposeIT.class.getClassLoader().getResourceAsStream(propertyFileName));
+            projectVersion = props.getProperty("project.version");
+            projectArtifactId = props.getProperty("project.artifactId");
```
Comment (danielezonca) [inline dmn-drools-springboot-metrics/src/test/java/org/kie/kogito/examples/ProjectMetadataProvider.java:33]: Same as above

**Review by danielezonca — APPROVED** [review]

**Pull request merged**